d20potz.py

```diff
@@ -480,6 +562,59 @@ async def cards_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
         return
 
 
+async def claims_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    chat_id = update.effective_chat.id
+    await print_all_claims(update, context, chat_id)
+
+async def claim_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    params = update.message.text.split()
+    chat_id = update.effective_chat.id
+    user_id = update.message.from_user.id
+
+    if len(params) < 2:
+        await context.bot.send_message(
+            chat_id=chat_id, text="Usage: /claim <player>"
+        )
+        return
+
+    player_name = params[1].lower()
+    if not await check_player_name(context, chat_id, player_name):
+        return   
+    
+    if is_player_claimed(chat_id, player_name):
+        if is_player_claimed_by_user(chat_id, player_name, user_id):
+            await context.bot.send_message(
```

Comment:
The logic is good but I think it's too strict.
Just allow claiming another character without unclaiming or checking if it was claimed already, it should be simpler.

a -> /claim heskan -> A claimed Heskan
b -> /claim vistra -> B claimed Vistra
a -> /claim vistra -> A claimed Vistra (Heskan is unclaimed and B does not own Vistra anymore)

Reply:
+